src/main/java/mekanism/common/item/interfaces/IJetpackItem.java

```diff
@@ -95,7 +96,9 @@ static ItemStack getActiveJetpack(LivingEntity entity) {
      */
     static boolean handleJetpackMotion(Player player, JetpackMode mode, BooleanSupplier ascendingSupplier) {
         Vec3 motion = player.getDeltaMovement();
-        if (mode == JetpackMode.NORMAL) {
+        if (mode == JetpackMode.ASCEND) {
+            player.setDeltaMovement(motion.x(), Math.min(motion.y() + 0.15D, 0.5D), motion.z());
```

Comment:
I think this is kind of pointless and shouldn't be added tbh.

Reply:
So I've been playing around with oriented for a while and in general I think it's an improvement. But I have found some situations where I'd rather it just be vertical so I can look up or down, like shafts. Also landing on single floating blocks is much easier and controlled when going up doesn't also mean changing your horizontal position.
Anyway, that's my opinion. I'm willing to change it however you feel best at the end of the day.
Or are you saying that oriented is pointless?

Reply:
Ah, I haven't looked yet at the changes to orientation so forgot that this might just be the old version.